Source/Scene/PostProcessCollection.js

```diff
@@ -0,0 +1,596 @@
+define([
+        '../Core/Check',
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/defineProperties',
+        '../Core/destroyObject',
+        '../Core/DeveloperError',
+        '../Shaders/PostProcessFilters/FXAA',
+        '../Shaders/PostProcessFilters/PassThrough',
+        '../ThirdParty/Shaders/FXAA3_11',
+        './PostProcess',
+        './PostProcessAmbientOcclusionStage',
+        './PostProcessBloomStage',
+        './PostProcessSampleMode',
+        './PostProcessTextureCache'
+    ], function(
+        Check,
+        defaultValue,
+        defined,
+        defineProperties,
+        destroyObject,
+        DeveloperError,
+        FXAAFS,
+        PassThrough,
+        FXAA3_11,
+        PostProcess,
+        PostProcessAmbientOcclusionStage,
+        PostProcessBloomStage,
+        PostProcessSampleMode,
+        PostProcessTextureCache) {
+    'use strict';
+
+    var fxaaFS =
+        '#define FXAA_QUALITY_PRESET 39 \n' +
+        FXAA3_11 + '\n' +
+        FXAAFS;
+
+    var stackScratch = [];
+
+    /**
+     * A collection of {@link PostProcess}es and/or {@link PostProcessComposite}s.
+     * <p>
+     * The texture modified by each post process is the texture rendered to by the scene or the texture rendered
+     * to by the previous post-process or composite in the collection.
+     * </p>
+     * <p>
+     * If the ambient occlusion or bloom post-processes are enabled, they will execute before all other post-processes.
+     * </p>
+     * <p>
+     * If the FXAA post-process is enabled, it will execute after all other post-processes.
+     * </p>
+     *
+     * @alias PostProcessCollection
+     * @constructor
+     */
+    function PostProcessCollection() {
+        this._processes = [];
+        this._activeProcesses = [];
+
+        this._fxaa = new PostProcess({
```

Comment:
I expected this class to be a more generic collection like [PrimitiveCollection](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Scene/PrimitiveCollection.js), but it looks like it has some scene rendering specific logic.  Let's determine if this is the right place for it, or if it is better separation of concerns for the FXAA/AO/bloom logic to be directly in `Scene`.

Reply:
Or maybe not if this is because uses want to access these right from the collection.

We might just update the reference doc to say not to create this collection yourself.